apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx

```diff
@@ -161,10 +161,70 @@ describe("StopPageRedesign", () => {
     ).not.toBeNull();
   });
 
-  it("should only render current alerts and not future alerts", () => {
+  it("should only render closures, shuttles, and supensions", () => {
+    const now = new Date();
+    const future1 = add(now, { days: 1 });
+    const alertsForRoute: Alert[] = [
+      {
+        informed_entity: {
+          entities: [{ route: "Test Route 2" }]
+        } as InformedEntitySet,
+        active_period: [[dateFormatter(now), dateFormatter(future1)]],
+        lifecycle: "new",
+        id: "000003",
+        header: "Test Alert The Walkway has spillage",
+        effect: "detour"
+      },
+      {
+        informed_entity: {
+          entities: [{ route: "Test Route 2" }]
+        } as InformedEntitySet,
+        active_period: [[dateFormatter(now), dateFormatter(future1)]],
+        lifecycle: "new",
+        id: "000004",
+        header: "Road Closed",
+        effect: "shuttle"
+      },
+      {
+        informed_entity: {
+          entities: [{ route: "Test Route 2" }]
+        } as InformedEntitySet,
+        active_period: [[dateFormatter(now), dateFormatter(future1)]],
+        lifecycle: "new",
+        id: "000005",
+        header: "Stop Closed",
+        effect: "stop_closure"
+      },
+      {
+        informed_entity: {
+          entities: [{ route: "Test Route 2" }]
+        } as InformedEntitySet,
+        active_period: [[dateFormatter(now), dateFormatter(future1)]],
+        lifecycle: "new",
+        id: "000006",
+        header: "Station Closed",
+        effect: "station_closure"
+      }
+    ] as Alert[];
+
+    jest
+      .spyOn(useAlerts, "useAlertsByRoute")
+      .mockReturnValue({ status: FetchStatus.Data, data: alertsForRoute });
+
+    render(<StopPageRedesign stopId="Test 1" />);
+
+    expect(screen.getByText(/Road Closed/)).toBeInTheDocument();
+    expect(screen.getByText(/Stop Closed/)).toBeInTheDocument();
+    expect(screen.getByText(/Station Closed/)).toBeInTheDocument();
+    expect(screen.queryByText(/The Walkway has spillage/)).toBeNull();
+  });
+
+  it("should only render current stop alerts and route alerts within 7 days", () => {
```

Comment:
We should add a test for the 0 days, as that was the original behavior.

Reply:
The first Alert in `alertsForStop` tests for this as it's start date is now

Reply:
I am confused then because that is not what I am seeing.  The first alert in this tests uses `past1` and `future1` for its times, after having remove the use of `now`.
![Screen Shot 2023-06-08 at 7 35 06 AM](https://github.com/mbta/dotcom/assets/2679229/052d8413-a01b-468e-b6e8-cb179c3a3166)

Reply:
0 days means a currently active alert. Having the active period be now(which it was originally) or in the past would mean it's currently active. Will switch back to now but am confused what the original comment was requesting.